calendar_v17/odoo_office365_calendar/models/calendar_event.py

```
@@ -0,0 +1,651 @@
+                end_date = recurrence.get("range", {}).get("endDate")
+
+                frequency_mapping = {
+                    "daily": "daily",
+                    "weekly": "weekly",
+                    "absoluteMonthly": "monthly",
+                    "absoluteYearly": "yearly",
+                }
+
+                
+
+                rrule = {
```

Ruff (0.11.9)

Local variable `rrule` is assigned to but never used

Remove assignment to unused variable `rrule`

(F841)